collectors/ebpf.plugin/ebpf.conf

```diff
@@ -19,11 +19,13 @@
 #
 # The eBPF collector enables and runs the following eBPF programs by default:
 #
+#  `cachestat`: Make charts for kernel functions related to page cache.
```

Comment:
Why do you use the `cachestat` word? I think we don't need to add the `stat` suffix. We can simply use `cache` or `pagecache`.

Reply:
We are using `cachestat` to keep compatibility with the names used by [`IOvisor`](https://github.com/iovisor/bcc/tree/master/tools), this was defined together our Product team (@manos-saratsis).